### PR TITLE
Alert logic docs

### PR DESCRIPTION
**Pull request by BobStolzberg**
2 docs for customers to deploy Alert Logic Threat Manager and Web Security Manager Partner Templates.

### CODE REVIEW

File: Ecosystem Partners/getting-started-with-alert-logic-threat-manager-partner-template.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+### Prerequisite
+- Access to the CenturyLink Cloud platform as an authorized user.
+- Identify a Network VLAN you want the Alert Logic Threat Manager to reside on
+
+### Postrequisite
+- If you want to access your Alert Logic Threat Manager over the internet, please perform the following tasks once your Alert Logic Threat Manager has been deployed to your account:
+
+1. Add a Public IP to your server
+![Add Public IP](https://t3n.zendesk.com/attachments/token/kObGC9P2IjP1ate0NexwFNiXz/?name=ip.jpg)
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-alert-logic-threat-manager-partner-template.md]: please don't link to zendesk images anymore. they should be added to this repo.

Comment (chrislittle) [inline Ecosystem Partners/getting-started-with-alert-logic-threat-manager-partner-template.md]: I'd recommend like all other KBs you place a link to our add public IP KB

File: Ecosystem Partners/getting-started-with-alert-logic-threat-manager-partner-template.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+10. Confirm the IP address.
+11. On the Alert Logic config screen, select 5.
+12. On the Ping an IP address screen, enter the IP address you want to ping.
+13. Confirm the data output.
+14. On the Alert Logic config screen, select 6.
+15. Navigate to http://<YOURVIRTUALAPPLIANCEIPADDRESS> in your web browser to confirm it is on the network.
+
+Please let me know if you have any questions or issues. Kindly send me a reply once the work has been completed and let us know the IP address of the Threat Manager VM.
+
+Thank you very much, Your_Name_Here
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-alert-logic-threat-manager-partner-template.md]: Is `Your_Name_Here` supposed to be here?

**Comment by BobStolzberg** [comment]
Can we please merge this pull request by COB?  Need the Alert logic docs Live!

**Pull request merged**